api/config/v1alpha1/configuration_types.go

```diff
@@ -108,6 +108,10 @@ type InternalCertManagement struct {
 	// Enable controls whether to enable internal cert management or not.
 	// Defaults to true. If you want to use a third-party management, e.g. cert-manager,
 	// set it to false. See the user guide for more information.
+	// This secret is mounted to the lws controller manager pod. The mount
```

Comment:
What secret do you mean here?

Reply:
Thanks for dropping your comment. I think, mentioning this behavior in this flag is not relevant. So that I've removed this comment entirely.